scripts/builder.php

```diff
@@ -8,7 +8,7 @@
 
 set_error_handler(
     function ($error, $message, $file, $line) use ($builder) {
-        if (error_reporting() !== 0) {
+        if (error_reporting() & $error) {
```

Comment:
"&" not "&&" with the bit operation? and not before (don't really see why it's needed)

Reply:
oups you answer to it